x/group/server/testsuite/suite.go

```diff
@@ -120,7 +120,8 @@ func (s *IntegrationTestSuite) SetupSuite() {
 	s.Require().NoError(err)
 	s.groupAccountAddr = addr
 
-	s.Require().NoError(s.bankKeeper.SetBalances(s.sdkCtx, s.groupAccountAddr, sdk.Coins{sdk.NewInt64Coin("test", 10000)}))
+	s.Require().NoError(s.bankKeeper.MintCoins(s.sdkCtx, minttypes.ModuleName, sdk.Coins{sdk.NewInt64Coin("test", 10000)}))
+	s.Require().NoError(s.bankKeeper.SendCoinsFromModuleToAccount(s.sdkCtx, minttypes.ModuleName, s.groupAccountAddr, sdk.Coins{sdk.NewInt64Coin("test", 10000)}))
```

Comment:
can't we use `simapp.FundAccount` here? (may be after this https://github.com/cosmos/cosmos-sdk/pull/9315 PR get merged)